Resources/public/js/views/services/plugins/ez-locationsloadplugin.js

```diff
@@ -49,10 +53,21 @@ YUI.add('ez-locationsloadplugin', function (Y) {
                 if (error) {
                     e.target.set('loadingError', true);
                 } else {
-                    e.target.set('locations', locations);
+                    if (e.loadPath) {
+                        Y.Array.each(locations, function (item) {
+                            item.loadPath(options, function (error) {
+                                if (error) {
+                                    e.target.set('loadingError', true);
+                                } else {
+                                    e.target.set('locations', locations);
```

Comment:
that's not gonna work, you need to use Y.Parallel to load the path for all Locations and set the result once everything is loaded.